src/pages/index.tsx

```diff
@@ -61,78 +66,21 @@ export default function Home({
   );
 }
 
-export const getStaticProps: GetStaticProps<PageProps> = async () => {
-  const content = await getAllContent();
+export const getStaticProps: GetStaticProps<{
+  content: ContentData;
+}> = async () => {
+  const content: ContentData = {
+    lectures: [],
+    resources: [],
+  };
+
+  (await getAllContent()).filter(item =>
```

Comment:
Also nitpick, but `forEach` might make more semantic sense here since we aren't actually filtering.

Reply:
great call, ill adjust